canonical-websites/www.ubuntu.com
# Version of server download

## Done
Added the three versions of the sign up in the download server thank you page.

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Change `default='true'` to the other version and check they match the designs in the issues and the [copy doc](https://docs.google.com/document/d/1S5zY7HN_Q8eQWch0mCEfLICfj8pW1bct9_wGArCNIqc/edit#)

## Issue / Card
Fixes: https://github.com/ubuntudesign/web-squad/issues/831
Fixes: https://github.com/ubuntudesign/web-squad/issues/834
Fixes: https://github.com/ubuntudesign/web-squad/issues/833

## Screenshots
![screenshot_2018-09-21 thanks for downloading ubuntu server ubuntu 2](https://user-images.githubusercontent.com/1413534/45872032-46c3a500-bd8f-11e8-9342-9749bec7fc6f.png)
![screenshot_2018-09-21 thanks for downloading ubuntu server ubuntu 1](https://user-images.githubusercontent.com/1413534/45872034-46c3a500-bd8f-11e8-86e3-9af87c786ee2.png)
![screenshot_2018-09-21 thanks for downloading ubuntu server ubuntu](https://user-images.githubusercontent.com/1413534/45872035-46c3a500-bd8f-11e8-9c4a-09c5f55a0c47.png)
